.gitlab-ci.yml

```diff
@@ -4,11 +4,14 @@ before_script:
    - pip install tox
 
 stages:
-   - build
-
-tox:
-   stage: build
-   script:
-      - tox
-   tags:
```

Comment:
Did we not need this docker tag?

Reply:
Not that I can see! https://gitlab.com/zcerza/ocs-ci/pipelines/70773478